specification/assets/gherkin/evaluation.feature

```diff
@@ -48,6 +48,19 @@ Feature: Flag evaluation
     Then the resolved object details value should be contain fields "showImages", "title", and "imagesPerPage", with values "true", "Check out these pics!" and 100, respectively
     And the variant should be "template", and the reason should be "STATIC"
 
+  Scenario: Passes evaluation details to after and finally hooks
+    When a hook is added to the client
+    And a boolean flag with key "boolean-flag" is evaluated with details and default value "false"
```

Comment:
this clashes with your other pr in writing, there are differences: 
```
 Given a Boolean-flag with key "metadata-flag" and a default value "true"
```

hence we should rework this to follow the same structure as the metadata pr